### PR TITLE
Created and updated docs for 1.21-5

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* Multiple, related patches that fix a security vulnerability with kubelet
+  * New patches
+    * **0009-EKS-PATCH-Pass-additional-flags-to-subpath-mount-to-avoid-flak.patch**
+    * **0010-EKS-PATCH-Add-missing-interface-method-in-mount_unsupported.go.patch**
+    * **0011-EKS-PATCH-Update-the-unit-tests-to-handle-mountFlags.patch**
+    * **0012-EKS-PATCH-Keep-MakeMountArgSensitive-and-add-a-new-signature-t.patch**
+  * About the patches
+    * Security fix for upstream Kubernetes
+    [issue #104980](https://github.com/kubernetes/kubernetes/issues/104980). 
+    The Kubernetes version used by this EKS-D release is v1.21.4, which is falls
```
Comment (kschumy) [inline docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md:34]:
```suggestion
    The Kubernetes version used by this EKS-D release is v1.21.4, which falls
```

File: docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    * **0009-EKS-PATCH-Pass-additional-flags-to-subpath-mount-to-avoid-flak.patch**
+    * **0010-EKS-PATCH-Add-missing-interface-method-in-mount_unsupported.go.patch**
+    * **0011-EKS-PATCH-Update-the-unit-tests-to-handle-mountFlags.patch**
+    * **0012-EKS-PATCH-Keep-MakeMountArgSensitive-and-add-a-new-signature-t.patch**
+  * About the patches
+    * Security fix for upstream Kubernetes
+    [issue #104980](https://github.com/kubernetes/kubernetes/issues/104980). 
+    The Kubernetes version used by this EKS-D release is v1.21.4, which falls
+    within the range of versions impacted by this security vulnerability.
+    * Patches are from the commit in upstream Kubernetes
```
Comment (kschumy) [inline docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md:36]:
```suggestion
    * Patches are from the commits in upstream Kubernetes 
```

File: docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    * **0012-EKS-PATCH-Keep-MakeMountArgSensitive-and-add-a-new-signature-t.patch**
+  * About the patches
+    * Security fix for upstream Kubernetes
+    [issue #104980](https://github.com/kubernetes/kubernetes/issues/104980). 
+    The Kubernetes version used by this EKS-D release is v1.21.4, which falls
+    within the range of versions impacted by this security vulnerability.
+    * Patches are from the commits in upstream Kubernetes 
+    [PR #104253](https://github.com/kubernetes/kubernetes/pull/104253), which 
+    should be part of Kubernetes v1.23. Upstream Kubernetes v1.21.5
+    [cherry-picked](https://github.com/kubernetes/kubernetes/pull/104347) the commit.
```
Comment (kschumy) [inline docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md:39]:
```suggestion
    [cherry-picked](https://github.com/kubernetes/kubernetes/pull/104347) the PR.
```

File: docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Contributor Shout Out
+
+Special thanks to [jonathan-conder-sm](https://github.com/jonathan-conder-sm) 
+for their contributions to this release, specifically their 
+thoroughly-investigated [issue](https://github.com/aws/eks-distro/issues/545) 
+and subsequent [PR](https://github.com/aws/eks-distro/issues/545) to 
+add a [patch](https://github.com/aws/eks-distro/blob/v1-21-eks-5/projects/kubernetes/kubernetes/1-21/patches/0010-PATCH-kubeadm-CoreDNS-permissions-for-endpointslices.patch)
+that the problem.
```
Comment (kschumy) [inline docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md:48]:
```suggestion
and subsequent [PR](https://github.com/aws/eks-distro/issues/545) 
adding a [patch](https://github.com/aws/eks-distro/blob/v1-21-eks-5/projects/kubernetes/kubernetes/1-21/patches/0010-PATCH-kubeadm-CoreDNS-permissions-for-endpointslices.patch)
that fixed the bug.
```

File: docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    [cherry-picked](https://github.com/kubernetes/kubernetes/pull/104347) the PR.
+
+## Contributor Shout Out
+
+Special thanks to [jonathan-conder-sm](https://github.com/jonathan-conder-sm) 
+for their contributions to this release, specifically their 
+thoroughly-investigated [issue](https://github.com/aws/eks-distro/issues/545) 
+and subsequent [PR](https://github.com/aws/eks-distro/issues/545) 
+adding a [patch](https://github.com/aws/eks-distro/blob/v1-21-eks-5/projects/kubernetes/kubernetes/1-21/patches/0010-PATCH-kubeadm-CoreDNS-permissions-for-endpointslices.patch)
+that fixed the bug.
```
Comment (kschumy) [inline docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md:48]:
```suggestion
that added a [patch](https://github.com/aws/eks-distro/blob/v1-21-eks-5/projects/kubernetes/kubernetes/1-21/patches/0010-PATCH-kubeadm-CoreDNS-permissions-for-endpointslices.patch)
and fixed the bug.
```

File: docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    [PR #104253](https://github.com/kubernetes/kubernetes/pull/104253), which 
+    should be part of Kubernetes v1.23. Upstream Kubernetes v1.21.5
+    [cherry-picked](https://github.com/kubernetes/kubernetes/pull/104347) the PR.
+
+## Contributor Shout Out
+
+Special thanks to [jonathan-conder-sm](https://github.com/jonathan-conder-sm) 
+for their contributions to this release, specifically their 
+thoroughly-investigated [issue](https://github.com/aws/eks-distro/issues/545) 
+and subsequent [PR](https://github.com/aws/eks-distro/issues/545)
```
Comment (abhay-krishna) [inline docs/contents/releases/1-21/5/CHANGELOG-v1-21-eks-5.md:46]:
```suggestion
and subsequent [PR](https://github.com/aws/eks-distro/pull/547)
```

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Pull request merged**